.github/workflows/push.yaml

```
@@ -0,0 +1,262 @@
+    name: Testing build for android
+    permissions:
+      contents: write
+    environment: TALAWA_ENVIRONMENT
+    runs-on: ubuntu-latest
+    needs: Flutter-Testing
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu' # See 'Supported distributions' for available options
+          java-version: '12.0'          
```

yamllint

[error] trailing spaces

(trailing-spaces)